rclpy/test/test_executor.py

```diff
@@ -270,7 +270,7 @@ def func():
         self.assertTrue(future.done())
         self.assertEqual('Sentinel Result', future.result())
 
-    def test_create_task_dependent_coroutines(self):
+    def test_create_task_fifo_order(self):
```

Comment:
this new test is similar with `test_create_task_dependent_coroutines` but it is the most simple case to check if the tasks are executed in FIFO order without dependent coroutines.